README.md

```diff
@@ -145,6 +145,10 @@ Check out [node-pg-pool](https://github.com/brianc/node-pg-pool) and [node postg
       <td>Boolean</td>
       <td>Whether to try SSL/TLS to connect to server</td>
     </tr>
+    <tr>
+      <td>defaultSort</td>
+      <td>boolean / string</td>
+      <td>Set to <code>false</code> to disable default sorting on <code>id</code> column(s), set to <code>numericIdOnly</code> to only apply to ID's with a number type <code>id</code>.</td>
```

Comment:
Nitpick: `column(s), set` -> `column(s). Set`
`apply to ID's` -> `apply to IDs`

Reply:
Thanks, corrected it.